samples/encryption/encryption-conventions/Core_5/Shared/EncryptionExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+#region ConfigureEncryption
```

Comment:
i think this region can be inside the method. the fact that u r using an extenion method is not really relevant?